business_requirement_earned_value/report/project_completion_report.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api, fields, tools
+
+
+class ProjectCompletionReport(models.Model):
+    """
+        Project Completion Report
+    """
+    _name = "project.completion.report"
+    # Database table should not be created, use init() instead
+    _auto = False
+    _description = "Project Completion Report"
+    # Field used for the Name
+    _rec_name = 'activity_name'
+
+    id = fields.Integer('ID', readonly=True)
+    partner_id = fields.Many2one(
+        'res.partner', 'Stakeholder', readonly=True)
+    br_id = fields.Many2one(
+        'business.requirement', 'Bus. Req.',
+        readonly=True, help="Business Requirement")
+    master_project_id = fields.Many2one(
+        'project.project', 'Master Project', readonly=True,
+        help="Master Project of the Business Requirement")
+    project_id = fields.Many2one(
+        'project.project', 'Project', readonly=True)
+    account_id = fields.Many2one(
+        'account.analytic.account', 'Analytic Account', readonly=True)
+    activity_type = fields.Selection(
+        [
+            ('task', 'Task'),
+            ('issue', 'Issue'),
+        ], 'Type', readonly=True,
+        help="Type is used to separate Tasks and Issues")
+    activity_id = fields.Char(
+        'Activity ID', readonly=True, help="Task ID or Issue ID")
+    activity_name = fields.Char(
+        'Activity Name', readonly=True, help="Task name or Issue name")
+    user_id = fields.Many2one(
+        'res.users',
+        'Assignee',
+        readonly=True,
+        help="Assignee is not necessarily the one who input the Timesheets")
+    activity_stage_id = fields.Many2one(
+        'project.task.type', 'Activity stage',
+        readonly=True, help="Activity Stage")
+    # FIXME if BR resource UoM is not hours, `qty` needs to be converted

```

Comment:
Add a note in the Roadmap section of the README:
`* if BR resource UoM is not hours, the 'qty' is incorrect (should be converted in the SQL view)`